deploy.sh

```diff
@@ -0,0 +1,38 @@
+# pkg="aws"
+# which $pkg > /dev/null 2>&1
+# if [ $? != 0 ];
+# then
+# echo "aws-cli installing..."
+
+# curl "https://s3.amazonaws.com/aws-cli/awscli-bundle.zip" -o "awscli-bundle.zip"
+# unzip awscli-bundle.zip
+# ./awscli-bundle/install -b ~/bin/aws
+# rm -f awscli-bundle.zip
+
+# echo "aws-cli installing has finished"
+# else
+# echo "aws-cli is already installed"
+# fi
+
+# echo "aws configuring..."
+# aws configure
+
+# npm install
+
+if [ "$1" == "prod" ];
```

Comment:
I would make the argument required to makes sure that nobody runs it by accident. Require to pass `prod` or `test` values.